src/Reflection/RequireExtension/RequireExtendsPropertiesClassReflectionExtension.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Reflection\RequireExtension;
+
+use PHPStan\Analyser\OutOfClassScope;
+use PHPStan\Reflection\ClassReflection;
+use PHPStan\Reflection\PropertiesClassReflectionExtension;
+use PHPStan\Reflection\PropertyReflection;
+use PHPStan\ShouldNotHappenException;
+use PHPStan\Type\VerbosityLevel;
+
+class RequireExtendsPropertiesClassReflectionExtension implements PropertiesClassReflectionExtension
```

Comment:
one extendsion for properties, one for methods? or combined in one class?

and another pair for require-implements?

Reply:
I think we can combine extends/implements into a single extension, but keep methods and properties as separate extensions.